### PR TITLE
1050: support LocationIndicatorActive property for CXP connector resource

### PR DESCRIPTION
**Pull request by lxwinspur**
Implement LocationIndicatorActive property for CXP connector resource

Upstream: 
Add Ports schema: https://gerrit.openbmc.org/c/openbmc/bmcweb/+/61090
Implementation of PortCollection schema in bmcweb: https://gerrit.openbmc.org/c/openbmc/bmcweb/+/61097

### CODE REVIEW

**Review by baemyung — COMMENTED** [review]
@lxwinspur 
Please add upstream web-link to each commit message.  It will help a lot later.
For example, the upstream for this Add Port schema seems https://gerrit.openbmc.org/c/openbmc/bmcweb/+/61090

**Comment by gtmills** [comment]
I am going to default to Myung, whenever he approves, I am good.

**Comment by lxwinspur** [comment]
> @lxwinspur Please add upstream web-link to each commit message. It will help a lot later. For example, the upstream for this Add Port schema seems https://gerrit.openbmc.org/c/openbmc/bmcweb/+/61090

@baemyung 
I have added upstream web-link, but the third submission has no upstream web-link.

File: redfish-core/include/generated/enums/port.hpp [hunk 1]
```diff
@@ -0,0 +1,237 @@
+#pragma once
```
Comment (gtmills) [inline redfish-core/include/generated/enums/port.hpp:1]: Why do we need this file?

**Comment by lxwinspur** [comment]
> > I have added upstream web-link,
> > I meant to add the link into the individual commit message so that we can see it from it directly from each commit (git show).
> 
> > but the third submission has no upstream web-link.
> > Are you planning to push into upstream later?

Yes, I will push a new patch to Gerrit, maybe next week

> > If so, wouldn't it better to have a WIP upstream draft submission now so that it can be remembered and referenced?

File: redfish-core/include/generated/enums/port.hpp (hunk 1 above)
Comment (lxwinspur) [reply redfish-core/include/generated/enums/port.hpp:1]: So sorry, I forgot to click the `submit` button yesterday :(

**Review by gtmills — APPROVED** [review]

**Pull request merged**